examples/topics/stats/pyramid.py

```diff
@@ -0,0 +1,57 @@
+''' A pyramid plot showing the age distribution of the male and female Titanic passengers using the `Bokeh` hbar glyph.
+
+.. bokeh-example-metadata::
+    :sampledata: titanic
+    :apis: bokeh.plotting.figure.hbar
+    :refs: :ref:`ug_topics_stats_pyramid`
+    :keywords: pyramid, hbar
+'''
+import numpy as np
+
+from bokeh.models import CustomJSTickFormatter, Label
+from bokeh.plotting import figure, show
+from bokeh.sampledata.titanic import data as df
+
+sex_group = df.groupby("sex")
+
+female_ages = sex_group.get_group("female")["age"].dropna()
+male_ages = sex_group.get_group("male")["age"].dropna()
+
+bin_width = 5
+bins = np.arange(0, 72, bin_width)
+m_hist, edges = np.histogram(male_ages, bins=bins)
+f_hist, edges = np.histogram(female_ages, bins=bins)
+
+colors = ("#055BB2", "#CB6805", "#808080")
```

Comment:
Where did these colors come from? would it be feasible to import some of the palettes at 

https://docs.bokeh.org/en/latest/docs/reference/palettes.html#accessible-palettes

and use colors direclty from those so that we also demonstrate usage of the accessible paletted?

Reply:
I just used colors traditionally associated with the genders. I guess I can the bokeh palettes as well...

Reply:
> I just used colors traditionally associated with the genders. I guess I can use the bokeh palettes as well...

We actually purposefully avoided using blue and pink (stereotypical color here) intentionally, I think we should avoid them in general in our examples.